inst/rmarkdown/templates/transmissibility/skeleton/skeleton.Rmd

```diff
@@ -304,14 +302,15 @@ Here we identify the key data needed in the analyses, including:
 ```{r}
 date_var <- "date"
 group_var <- "region"
-count_var <- "n"
+# Leave count_var as NULL if the data is really a linelist / patient-level data.
```

Comment:
This will have to be included in the documentation for users as well

Reply:
Aren't the comments in the report the documentation for users?

Reply:
I mean in vignettes as well that explain in more detail how to interact with the templates

Reply:
Got it! As far I know though, we don't really have this kind of info in the vignette at the moment. Do you think it can wait for a later PR on this specifically? Or do you believe it has to be part of this PR?

Reply:
Yes, sounds great to me. Let's merge this PR then.